src/openrct2-ui/input/MouseInput.cpp

```diff
@@ -526,7 +526,10 @@ static void InputViewportDragBegin(rct_window& w)
     _ticksSinceDragStart = 0;
     auto cursorPosition = context_get_cursor_position();
     gInputDragLast = cursorPosition;
-    context_hide_cursor();
+    if (!gConfigGeneral.invert_viewport_drag)
```

Comment:
I think you forget to remove this if statement, same for the other ones below. Just remove the `context_hide_cursor()` call in this function and that should be it.

Reply:
Doesn't removing the call entirely also remove it for right-mouse panning (or non-inverted dragging)? I'd rather keep it as-is when inverted right-mouse dragging is disabled.

Reply:
You're right, I personally never drag the screen around (only use the keyboard) so I missed that nuance. Thanks!

In that case, @beam41, please fix the compile errors by replacing `invert_viewport_drag` with `InvertViewportDrag`.